src/settings-ui/Settings.UI/Views/AwakePage.xaml

```diff
@@ -37,37 +39,48 @@
                     Severity="Informational" />
 
                 <controls:SettingsGroup
-                    x:Uid="Awake_Behavior_GroupSettings"
+                    x:Uid="Awake_BehaviorSettingsGroup"
                     IsEnabled="{x:Bind Mode=OneWay, Path=ViewModel.IsEnabled}">
 
                     <labs:SettingsCard
-                        x:Uid="Awake_Mode"
+                        x:Uid="Awake_ModeSettingsCard"
                         HeaderIcon="{ui:FontIcon FontFamily={StaticResource SymbolThemeFontFamily}, Glyph=&#xE945;}">
                         <ComboBox
                             MinWidth="{StaticResource SettingActionControlMinWidth}"
                             SelectedIndex="{x:Bind Path=ViewModel.Mode, Mode=TwoWay, Converter={StaticResource AwakeModeToIntConverter}}">
-                            <ComboBoxItem x:Uid="Awake_NoKeepAwake" />
-                            <ComboBoxItem x:Uid="Awake_IndefiniteKeepAwake" />
-                            <ComboBoxItem x:Uid="Awake_TemporaryKeepAwake" />
+                            <ComboBoxItem x:Uid="Awake_NoKeepAwakeSelector" />
+                            <ComboBoxItem x:Uid="Awake_IndefiniteKeepAwakeSelector" />
+                            <ComboBoxItem x:Uid="Awake_TemporaryKeepAwakeSelector" />
+                            <ComboBoxItem x:Uid="Awake_ExpirableKeepAwakeSelector" />
                         </ComboBox>
                     </labs:SettingsCard>
 
                     <labs:SettingsCard
-                        x:Uid="Awake_TimeBeforeAwake"
+                        x:Uid="Awake_ExpirationSettingsCard"
+                        HeaderIcon="{ui:FontIcon FontFamily={StaticResource SymbolThemeFontFamily}, Glyph=&#xEC92;}"
+                        Visibility="{x:Bind ViewModel.IsExpirationConfigurationEnabled, Mode=OneWay}">
+                        <StackPanel Orientation="Horizontal">
```

Comment:
@dend @niels9001 These controls stay horizontal and are not very responsive.
![image](https://user-images.githubusercontent.com/65828559/226484664-653ecaaa-2579-4326-b854-4d7b5fa2fc4d.png)